account_controllers/controllers.js

```diff
@@ -123,27 +132,65 @@ module.exports = {
       }
     },
 
+    editVehicle: async (req, res) => {
+      try {
+        const { id, make, model, color, license_plate_number } = req.body;
+        const { error } = await supabase
+          .from('cars')
+          .update({
+            make,
+            model,
+            color,
+            license_plate_number,
+          })
+          .eq('id', id);
+
+        if (error) {
+          console.error('Error editing vehicle:', error);
+          return res.sendStatus(500);
+        }
+
+        res.status(200).send('Successfully edited vehicle');
+      } catch (error) {
+        console.error('Error editing vehicle:', error);
+        res.status(500).send('Internal Server Error');
+      }
+    },
+
     getCurrentReservations: async (req, res) => {
       try {
         const { id } = req.params;
+        const today = new Date();
+        today.setHours(0, 0, 0, 0);
+
+        const formattedToday = formatDate(today);
+
         const { data, error } = await supabase
           .from('reservations')
           .select(
             `
+            id,
             parking_spot_id,
             date,
             time,
-            garages ( name, address, city, state, country, zip)
+            garages (name, address, city, state, country, zip)
           `
           )
-          .eq('user_id', id);
+          .eq('user_id', id)
+          .in('status', ['reserved', 'checked-in'])
+          .order('date', { ascending: true })
+          .order('time', { ascending: true })
+          .eq('date', formattedToday);
+
```

Comment:
summary: retrieves all reservations for the current date today as constructed through luxon and returns the list ordered by date and time